pkg/restapi/operation/operations.go

```diff
@@ -340,6 +346,48 @@ func (c *Operation) onboardUser(id string) (*user.Profile, error) {
 	return userProfile, nil
 }
 
+func (c *Operation) handleBootstrapDataRequest(w http.ResponseWriter, r *http.Request) {
+	handle := r.URL.Query().Get("up")
```

Comment:
What is "up"?

Reply:
@DRK3 stands for "user profile", and is being added in #43